src/main/routes/health.ts

```diff
@@ -39,7 +39,9 @@ export default (app: Application): void => {
   const healthCheckConfig = {
     checks: {
       casBuyerUi: healthCheck.raw(() => healthCheck.up()),
-      agreementService: healthCheck.web(agreementsService.url.statusURL(), genericConfig),
+      agreementService: healthCheck.raw(() => {
+        return agreementsService.api.getAgreementsServiceHealth() ? healthCheck.up() : healthCheck.down();
+      }),
```

Comment:
If the call to the agreement service health endpoint status will be 200 when the AS is up then you we should do something like this

```suggestion
      agreementService: healthCheck.raw(async () => {
        return (await agreementsService.api.getAgreementsServiceHealth()).status === FetchResultStatus.OK ? healthCheck.up() : healthCheck.down();
      }),
```

Reply:
Thanks, this helped!  Implemented :)